### ISSUE TITLE
Jsonschema>=3.0.0 support in releases?

### ISSUE THREAD

**Issue by daisieh** [zalando/connexion#1345]
### Description

I see in #1050 that support for jsonschema >=3.0.0 was merged into master back in 2019, but was never part of any PyPI release. Is there a reason for that? It's causing problems for my package that requires both openapi-core (which requires jsonschema>=3.0.0) and connexion requiring less than 3.0.0, and it seems like one should be able to have a project that uses both...

Thanks!

**Comment by RichardBruskiewich** [comment]
I totally agree. This is a total show stopper in the real work of software production. I've hit a wall with this.

**Comment by RobbeSneyders** [comment]
Hi @daisieh @RichardBruskiewich 
As far as I can tell, [these changes are part of the 2.7 release](https://github.com/zalando/connexion/blob/2.7.0/setup.py). Installing `connexion==2.7.0` also installs `jsonschema==3.2.0` for me. 
Closing this for now, but feel free to reopen if you can provide some more info indicating that this is a connexion issue.

**Comment by RichardBruskiewich** [comment]
Hello @RobbeSneyders,
I don't know... I ultimately had to do a deep dive on Connexion to make it work for my application. I forked connexion to apply a  mix  of unmarked patches to make it work  for my use cases: **https://github.com/STARInformatics/connexion.git@fix-urlencoded-body-parameter-parsing**. I don't recall if this issue related to my effort  at that time to make things  work, or  to an earlier  attempt.  I have substituted this fork into my system but will likely need to review if connexion has been fixed thus  the main release, usable  for my  system.
Cheers
Richard
